tests/integration/interval/test_interval.py

```diff
@@ -0,0 +1,71 @@
+import dash
```

Comment:
All legacy integration tests have been ported to use the dash_dcc fixture instead